class.jetpack-gutenberg.php

```diff
@@ -185,9 +185,13 @@ public static function set_extension_available( $slug ) {
 	 *
 	 * @param string $slug Slug of the extension.
 	 * @param string $reason A string representation of why the extension is unavailable.
+	 * @param array  $details A free-form array containing more information on why the extension is unavailable.
 	 */
-	public static function set_extension_unavailable( $slug, $reason ) {
-		self::$availability[ self::remove_extension_prefix( $slug ) ] = $reason;
+	public static function set_extension_unavailable( $slug, $reason, $details = array() ) {
+		self::$availability[ self::remove_extension_prefix( $slug ) ] = array(
+			'reason'  => $reason,
+			'details' => $details,
+		);
```

Comment:
Do you know if there is anything outside Jetpack expecting this still to return a string instead of an array?

Reply:
This is just the internal implementation; we're not changing the type of the API response of `unavailable_reason`, it continues to be a string. We're only adding another `details` field, which by established practice (and even some documentation on A8c REST APIs) should not require a version bump:

![image](https://user-images.githubusercontent.com/96308/60496580-ec1f8800-9cd4-11e9-8a68-e23412338f2b.png)